### ISSUE TITLE
Actual result returned by z_shieldcoinbase does not match cli help text

### ISSUE THREAD

**Issue by arcalinea** [zcash/zcash#2777]
### Expected behavior:
`zcash-cli help z_shieldcoinbase` tells users to expect this result:

``` Result:
{
  "operationid": xxx          (string) An operationid to pass to z_getoperationstatus to get the result of the operation.
  "shieldedUTXOs": xxx        (numeric) Number of coinbase utxos being shielded.
  "shieldedValue": xxx        (numeric) Value of coinbase utxos being shielded.
  "remainingUTXOs": xxx       (numeric) Number of coinbase utxos still available for shielding.
  "remainingValue": xxx       (numeric) Value of coinbase utxos still available for shielding.
}
````

### Discrepancy in actual behavior:
The actual result returned by `z_shieldcoinbase`:
```
{
  "remainingUTXOs": 0,
  "remainingValue": 0.00000000,
  "shieldingUTXOs": 1,
  "shieldingValue": 10.00000000,
  "opid": "opid-d6573e4e-4c44-4d3f-b351-d719eda5a522"
}
```

I ran into this when I got an error while trying to parse the result, I expected the field "operationid" but `res['operationid']` did not work

**Comment by arcalinea** [comment]
Closed by #2724

**Comment by daira** [comment]
I hadn't realized that we were inconsistent in the naming of this field between different operations. That's unfortunate.
